client/components/deposits-overview/index.tsx

```diff
@@ -144,50 +149,54 @@ const DepositsOverview: React.FC = () => {
 				</>
 			) }
 
-			<CardFooter className="wcpay-deposits-overview__footer">
-				<Button
-					variant="secondary"
-					href={ getAdminUrl( {
-						page: 'wc-admin',
-						path: '/payments/deposits',
-					} ) }
-					onClick={ () =>
-						wcpayTracks.recordEvent(
-							wcpayTracks.events
-								.OVERVIEW_DEPOSITS_VIEW_HISTORY_CLICK
-						)
-					}
-				>
-					{ __(
-						'View full deposits history',
-						'woocommerce-payments'
+			{ ( hasRecentDeposits || canChangeDepositSchedule ) && (
```

Comment:
This is a minor change, recommend using the "Hide whitespace" option in the GH diff viewer